packages/lib-classifier/src/plugins/tasks/TextTask/models/TextTask.js

```diff
@@ -0,0 +1,28 @@
+import { getRoot, types } from 'mobx-state-tree'
+import TextAnnotation from './TextAnnotation'
+
+const TextTask = types.model('TextTask', {
+  help: types.optional(types.string, ''),
+  instruction: types.string,
+  required: types.optional(types.boolean, false),
+  taskKey: types.identifier,
+  text_tags: types.array(types.string),
+  type: types.literal('text')
+})
+  .views(self => ({
+    get annotation () {
+      const { currentAnnotations } = getRoot(self).classifications
```

Comment:
I don’t like the coupling with the classifications model here. The annotation could come from a classification or a drawing mark.

Reply:
Moved to #1239.